docs/docs/examples/guides/07-orbbec-guide.md

```diff
@@ -0,0 +1,121 @@
+Linux devices for now.
+:::
+
+In order to also ensure the camera can be accessed from within the container, the `/run/udev/` volume
+has to be mounted. In the AICA Launcher window, expand the **Advanced Settings** in the bottom, select
+**Add a volume mount** and define the volume, as shown in the following image:
+
+<div class="text--center">
+  <img src={orbbecMountedVolume} alt="Mounted udev volume" />
+</div>
+
+Select **Launch AICA Studio** to proceed. 
+
+## Creating and running an application
+
+Start by creating a new application. 
+
+1. By default, the new application contains a hardware interface, which can later be used to also
+connect and include a robot. For now, it is not necessary, so it can be removed. 
+2. Click on the Add button on the top right, and locate the **Orbbec Camera** component. Click to add
+to the graph. 
+3. Finally, connect the start block to the component.
+
+<div class="text--center">
+  <img src={orbbecNewApp} alt="Creating a new Orbbec Camera component" />
+</div>
+
+4. Select **Play** to start the application. 
+5. Once the component appears loaded, click on the gear icon on the bottom right and select
+**Launch RViz**. This will allow to check the live camera feed. 
+6. Once RViz launches, select _Add > By topic > /color_image > Image_. This adds a panel that shows the
+live color image. The depth image can also be found under _/depth_image > Image_.
+
+<div class="text--center">
+  <img src={orbbecRvizColor} alt="Starting and checking camera live stream" />
+</div>
+
+## Parametrizing Orbbec camera
+
+Click on the small gear icon on the `Orbbec Camera` block to view and edit the available parameters.
+
+<div class="text--center">
+  <img src={orbbecCameraParameters} alt="Basic Orbbec camera parameters" />
+</div>
+
+Hovering over the exclamation marks next to the names shows a detailed decription of the parameter. 
+Let's explain some of these here:
+
+- `Color/Depth Width/Height/FPS`: these refer to the resolution and frame rate of the color and depth
+images. Keep in mind that only specific pairs of integer values apply here. For more information 
+check the camera's documentation. The Orbbec Viewer can also come in handy here, as it can be used 
+to see what profiles are available.
+- `Enable Alignment`: boolean that activates the spatial alignment of the depth image to the
+corresponding color image. Generates a depth image with the same size as the color, with depth
+expressed in the color camera coordinate system. 
+- `Enable Threshold Filter`: boolean that activates a threshold filter in the depth image. Activation
+of the filter allows the optional definition of some additional parameters -if they are not defined, the
+default values will be used. To see what parameters can be defined, hover over the exclamation mark. To 
+avoid cluttering the parameter tab of the component, these parameters can only be defined in the YAML.
```

Comment:
I would not necessarily phrase this like that, more like;
Enable XX Filter - if activated, optional parameters can/should be directly defined in the YAML editor to configure the filter parameters